miso-web/src/main/webapp/pages/adminsub.jsp

```diff
@@ -45,6 +45,7 @@
         <li><a href="<c:url value="/miso/arrayruns"/>">Array Runs</a></li>
         <li><a href="<c:url value="/miso/boxes"/>">Boxes</a></li>
         <li><a href="<c:url value="/miso/instruments"/>">Instruments</a></li>
+        <li><a href="<c:url value="/miso/storagelocations"/>">Storage Locations</a></li>
```

Comment:
Can we move this to be right next to Boxes? Maybe by moving `Instruments` above Boxes and below `Array Runs`...

Reply:
Also, what do you think about calling it `Freezers & Rooms`? (Makes it less abstract)

Reply:
Sure. Fixed.